types/src/sector/mod.rs

```diff
@@ -17,6 +17,10 @@ use vm::ActorID;
 
 pub type SectorNumber = u64;
 
+/// The maximum assignable sector number.
+/// Raising this would require modifying our AMT implementation.
+pub const MAX_SECTOR_NUMBER: SectorNumber = SectorNumber::MAX;
```

Comment:
This should be the max number from the Amt, not u64 max. The max for the Amt is slightly less (Max i64), probably best to just use the value directly from the Amt to future proof though